external/avro/src/main/scala/org/apache/spark/sql/avro/AvroDeserializer.scala

```diff
@@ -62,10 +62,10 @@ private[sql] class AvroDeserializer(
   private lazy val decimalConversions = new DecimalConversion()
 
   private val dateRebaseFunc = DataSourceUtils.creteDateRebaseFuncInRead(
-    datetimeRebaseMode, "Avro")
+    datetimeRebaseMode, "Avro")(_, _)
```

Comment:
@AngersZhuuuu, seems like this commit was sneaked in mistakenly in this PR

Reply:
> @AngersZhuuuu, seems like this commit was sneaked in mistakenly in this PR

Yea..when work on other pr, forgot to change branch

Reply:
Fixed